aiida/cmdline/commands/cmd_group.py

```diff
@@ -306,9 +318,13 @@ def group_list(
     for group in result:
         table.append([projection_lambdas[field](group[0]) for field in projection_fields])
 
-    if not all_types:
-        echo.echo_info('If you want to see the groups of all types, please add -a/--all-types option')
-    echo.echo(tabulate(table, headers=projection_header))
+    if not all_entries:
+        echo.echo_info('to show groups of all types, use the `-a/--all-types` option.')
```

Comment:
I think the lower case here is okay, since it is preceded by `Info:`. I don't think we have agreed on a fixed style guide here. Maybe we should and then update all commands consistently, but I will leave this for now in this PR.

Reply:
> (...) Maybe we should and then update all commands consistently (...)

I'd like that.

Reply:
While the capitalization-suggestion has been overruled, the second suggestion here still stands. And checking `options.ALL` it seems to not only be a suggestion, but a fix.

Reply:
ah, apologies, overlooked the second part of the suggested change. Have updated it